core/go/internal/keymanager/keymanager_rpc_test.go

```diff
@@ -50,6 +51,78 @@ func TestRPCLocalDetails(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, "m/44'/60'/1'/0/0", resolvedKey.KeyHandle)
 
+	/*
+			       Keys used for testing
+		    |---------------------------------|
```

Comment:
Tiny nit visible in Github UI - looks like one of the tabs ended up as spaces on just this line

Reply:
Interestingly in my local environment this shows fine. I too noticed it does not appear correctly in the PR on GitHub. I re-wrote that comment, hopefully that will address the formatting issue.